redis/db/redisbp/monitored_client.go

```diff
@@ -97,7 +107,7 @@ func (cc *ClusterClient) Wait(ctx context.Context, args WaitArgs) (replicas int6
 // redisbp.SpanHook attached.
 func NewMonitoredClusterClient(name string, opt *redis.ClusterOptions) *ClusterClient {
 	client := redis.NewClusterClient(opt)
-	client.AddHook(SpanHook{ClientName: name})
+	client.AddHook(SpanHook{ClientName: name, Type: "cluster", Deployment: getDeploymentType(opt.Addrs[0])})
```

Comment:
Note that I don't have the DB here

Reply:
I would slightly prefer to split the `SpanHook` fields into one-per-line and explicitly add `Database: "", // we don't have this info for cluster clients`.